examples/docker-compose/docker-compose-community-nginx.yaml

```diff
@@ -48,13 +47,14 @@ services:
       - pgdata:/var/lib/postgresql/data
 
   nginx:
-    container_name: nginx
-    build: ./nginx
+    image: nginx:1.17
```

Comment:
Why not use the latest version of Nginx? Or at least something closer to the latest version available (nginx-1.21.1).

Reply:
I copy&paste some config 😅  I will use the version 1.21